example_job_cancel_from_client_test.go

```diff
@@ -38,19 +40,14 @@ func (w *SleepingWorker) Work(ctx context.Context, job *river.Job[CancellingArgs
 func Example_jobCancelFromClient() {
 	ctx := context.Background()
 
-	dbPool, err := pgxpool.NewWithConfig(ctx, riverinternaltest.DatabaseConfig("river_test_example"))
+	jobChan := make(chan int64)
```

Comment:
unintentional relocation? IMO it makes more sense down near where it's used

Reply:
Yeah agreed. I just moved it back.